packages/mobile/src/send/SendAmount.tsx

```diff
@@ -91,11 +90,28 @@ export const sendAmountScreenNavOptions = ({
 }
 
 function SendAmount(props: Props) {
-  const dispatch = useDispatch()
+  const { t } = useTranslation(Namespaces.sendFlow7)
+
+  const [amount, setAmount] = useState('')
+  const [reviewButtonPressed, setReviewButtonPressed] = useState(false)
+  const [reviewButtonInnerElement, setReviewButtonInnerElement] = useState<string | JSX.Element>(
+    t('global:review')
+  )
```

Comment:
nit: from what I see, this state could be removed and we could derive it from `reviewButtonPressed` and `recipientVerificationStatus`.

Reply:
Good point. That will save us a rerender.